### PR TITLE
Catpeople no longer have a monopoly on being snowflakes

### PR DESCRIPTION
**Pull request by SomeguyManperson**
image

:cl:  
tweak:  expands whitelist for pet collars as a joke
/:cl:

### CODE REVIEW

**Comment by LoliconSlayer** [comment]
This doesn't have the fuck label and I feel that should be changed

**Comment by Sir-Lagsalot** [comment]
This PR was made specifically so Lelf could more easily torment me and I for one do NOT approve

**Comment by Mqiib** [comment]
Something something cuff/leg cuff/collar new shitsec meta

**Comment by SomeguyManperson** [comment]
nobody even uses neck grabs lol

**Comment by SCortesBar** [comment]
Where is the UwU label

**Comment by boodaliboo** [comment]
Wait you can grab a neck

**Comment by LewdElf** [comment]
I presume the grab neck refers to one of the final stages of aggressive grabbing. First you grab someone. then you begin to restrain them. One more, you pull them to you. Another, you choke them. Or something like that.

**Comment by SCortesBar** [comment]
> 
> 
> Wait you can grab a neck

I presume the grab neck refers to one of the final stages of aggressive grabbing. First you grab someone. then you begin to restrain them. One more, you pull them to you. Another, you choke them. Or something like that.

**Comment by Potototatoh** [comment]
great very good and wholesome and good

File: code/modules/clothing/neck/_neck.dm [hunk 1]
```diff
@@ -199,11 +199,11 @@
...
-	if(ishuman(M))  // yogs start - lets catpeople wear collars
+	if(ishuman(M))
 		var/mob/living/carbon/human/T = M
-		if(iscatperson(T))
-			return TRUE
-		return FALSE // yogs end
+		if(!ishumanbasic(T))
+			return ..()
+		return FALSE
 	return ..()
```
Comment (alexkar598) [inline code/modules/clothing/neck/_neck.dm]:
```DM
	if(!ishumanbasic(T))
		return ..()
	return FALSE
```
Replace this whole proc with this

Comment (SomeguyManperson) [reply code/modules/clothing/neck/_neck.dm]: does that check not also occur for trying to equip collars to animals

Comment (SomeguyManperson) [reply code/modules/clothing/neck/_neck.dm]: or I might be retarded

**Comment by ThatGuyPhil** [comment]
one step closer to the electrocollar

**Comment by alexkar598** [comment]
Make it for humans too!!!!

**Comment by AsV9** [comment]
why though

**Pull request merged**